File: python/bmi.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def main(args):
    m = float(input("Podaj masę:"))
    print("Masa", m)

    w = float(input("Podaj wzrost:"))
    print("Wzrost: ", w)

    c = m / (w * w)
    print("BMI: ", c)

    if c < 18.5:
        print("Niedowaga")
    if c >= 25:
        print("Nadwaga")
    if c >= 30:
        print("Otyłość")
    else:
        print("Norma")

    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp_kl2/bmi.cpp
/*
 * kalkulator.cpp
*/


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{

    float m, w, c;
    m=w=c=0;
    cout << "Podaj masę: " << endl;
    cin >> m;
    cout << "Podaj wzrost w metrach: " << endl;
    cin >> w;
    c = m/(w*w);
    
    cout << "BMI: " << c << endl;
    
    if (c < 18.5) 
    {
    cout << "Niedowaga" << endl;
    }
    else if (c>= 25)
      cout << "Nadwaga " << endl;
    else if (c>= 30)
      cout << "Otyłość" << endl;
    else if ( c>= 18.5 || c<=24.9)
      cout << "Norma" << endl;
    
    return 0;
    
}

